### DISCUSSION TITLE
Roadmap for upgrading extensions to v3?

### DISCUSSION THREAD

**Discussion by pelikhan** [gatsbyjs/gatsby#29963] [Help]
Hi team,

Are you working on upgrading some of the plugins for v3? It would be nice to know which one will be upgraded and which ones will need help.

Also, it would be great to support "v3" compatible filter in the plugins list.

**Comment by LekoArts** [comment]
Hi!

Our own plugins in this repository here are compatible with v3. If you mean community plugins I'm afraid we can only assist with it and the maintainers are responsible with upgrading the packages accordingly.

**Reply by pelikhan** [reply]
Right, app compat over this large amount of community projects is tricky.

So right it's going to be a frustrating hit-or-miss on the plugins as there is no info from the list that it'll work in v3.

**Comment by pelikhan** [comment]
So i am digging a bit further on these errors. There are the warning i get:
``
success open and validate gatsby-configs - 0.276s
warn Plugin gatsby-transformer-plaintext is not compatible with your gatsby version 3.0.1 - It requires gatsby@^2.0.15
warn Plugin gatsby-plugin-react-helmet is not compatible with your gatsby version 3.0.1 - It requires gatsby@^2.0.0
warn Plugin gatsby-remark-embedder is not compatible with your gatsby version 3.0.1 - It requires gatsby@^2.20.0
warn Plugin gatsby-plugin-webpack-bundle-analyser-v2 is not compatible with your gatsby version 3.0.1 - It requires gatsby@^2.0.0
```

``gatsby-plugin-react-helmet`` is an official plugin. It is referenced by ``    "gatsby-plugin-react-helmet": "^4.0.0",`` in package.json

**Reply by pelikhan** [reply]
it seems that gatsby-plugin-react-helmet is at 4.0.0 on npm, not 4.1

**Comment by pragmaticpat** [comment]
@pelikhan - thanks for the question, and for the suggestion (identifying whether a plugin is V3 compatible yet). This is a key topic internally, and we're still working on the specifics.

From a basic standpoint, and to echo your initial comment, when looking at the current [Plugins page](https://www.gatsbyjs.com/plugins), it'd be very useful for users to be able to quickly determine the compatibility of a given plugin. Specifically "V3 Ready!" or the like.

**Reply by DanielTamkin** [reply]
Just spent the last hour trying to update a plugin that previously worked fine, only to find within a buried repo issue a mention that their was no support for v3 yet. Would love to see blanket tags on the plugin page, that'd greatly reduce headaches for sure.

**Comment by Geerodge** [comment]
Any news on an easy way to check if a plugin is compatible with Gatsby v3? I've got quite a few warnings from the following plugins:

- gatsby-plugin-react-helmet
- gatsby-source-filesystem
- gatsby-plugin-styled-components
- gatsby-transformer-sharp
- gatsby-plugin-manifest
- gatsby-plugin-google-gtag
- gatsby-plugin-react-helmet
- gatsby-source-filesystem
- gatsby-plugin-styled-components
- gatsby-transformer-sharp
- gatsby-plugin-manifest
- gatsby-plugin-google-gtag

Should I just check on their GitHub repos?
